community/examples/slurm-gcp-v5-ubuntu2004.yaml

```diff
@@ -0,0 +1,85 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+
+blueprint_name: slurm-gcp-v5-ubuntu2004
+
+vars:
+  project_id:  ## Set GCP Project ID Here ##
+  deployment_name: slurm-gcp-v5
+  region: us-central1
+  zone: us-central1-c
+  slurm_cluster_name: ghpc
+  source_image_family: "schedmd-v5-slurm-22-05-2-ubuntu-2004-lts"
+  source_image_project: "projects/schedmd-slurm-public/global/images/family"
```

Comment:
Is this just for clarity? It shouldn't be needed if I recall correctly. The source_image_project will default to this regardless of whether the image family has been set.

I actually like including it, but wanted to check if that was the purpose.

Reply:
Correct. That is the default value for source_image_project, but I felt this would make it more clear that this works because the project has the right value.